solutions/USACO open16-splitting-the-field.mdx

```diff
@@ -92,238 +92,65 @@ From the analysis:
 > Note that it is also possible to dispense with the binary search trees entirely and just keep running mins and maxes in $y$.
 
```

Comment:
Maybe we should add some more explanation for this solution?

Reply:
We probably should, but that's something for another PR(?).